### ISSUE TITLE
Title style

### ISSUE THREAD

**Issue by Mannydefreitas7** [nerdsupremacist/FancyScrollView#7]
Hello! Thanks for the great work!
Just one tiny issue: Could you have the title property be a text view instead of a string? So we can use custom font style.
Let me know

**Comment by PetreHoffmann** [comment]
Yes this will be Great. Only Black or White is not helpful in some situations

**Comment by PetreHoffmann** [comment]
I added textColor property to manage the text color. I need to make some test to see if your proposal works without bugs before I push the changes
